src/Zoom/README.md

```diff
@@ -0,0 +1,42 @@
+# Zoom
+
+```bash
+composer require socialiteproviders/zoom
+```
+
+## Installation & Basic Usage
+
+Please see the [Base Installation Guide](https://socialiteproviders.com/usage/), then follow the provider specific instructions below.
+
+### Add configuration to `config/services.php`
+
+```php
+'zoom' => [
+  'client_id' => env('ZOOM_CLIENT_ID'),
+  'client_secret' => env('ZOOM_CLIENT_SECRET'),
+  'redirect' => env('ZOOM_REDIRECT_URI')
+],
+```
+
+### Add provider event listener
+
+Configure the package's listener to listen for `SocialiteWasCalled` events.
+
+Add the event to your `listen[]` array in `app/Providers/EventServiceProvider`. See the [Base Installation Guide](https://socialiteproviders.com/usage/) for detailed instructions.
+
+```php
+protected $listen = [
+    \SocialiteProviders\Manager\SocialiteWasCalled::class => [
+        // ... other providers
+        'SocialiteProviders\\Zoom\\ZoomExtendSocialite@handle',
+    ],
+];
+```
+
+### Usage
+
+You should now be able to use the provider like you would regularly use Socialite (assuming you have the facade installed):
+
+```php
+return Socialite::driver('zoom')->redirect();
```

Comment:
Could you add the returned fields here?
Have a look at some other providers for examples :)